src/chocolatey.resources/helpers/functions/Install-ChocolateyShortcut.ps1

```diff
@@ -108,12 +127,39 @@ directoy, an icon to be used for the shortcut, along with a description and argu
 		if ($description) {
 		  $lnk.Description = $description
 		}
-	   
+    if ($WindowStyle) {
+      $lnk.WindowStyle = $WindowStyle
+    }
+
 	    $lnk.Save()
-		
+
 		Write-Debug "Shortcut created."
+
+    [System.IO.FileInfo]$Path = $shortcutFilePath
+
+    If ($RunAsAdmin) {
+      $TempFileName = [IO.Path]::GetRandomFileName()
+      $TempFile = [IO.FileInfo][IO.Path]::Combine($Path.Directory, $TempFileName)
+      $Writer = New-Object System.IO.FileStream $TempFile, ([System.IO.FileMode]::Create)
+      $Reader = $Path.OpenRead()
+      While ($Reader.Position -lt $Reader.Length) {
+        $Byte = $Reader.ReadByte()
+        If ($Reader.Position -eq 22) {$Byte = 34}
+        $Writer.WriteByte($Byte)

```

Comment:
You might leave a comment with an explanation or better, a link to an explanation of this.